### ISSUE TITLE
The Resource Collection Slideshow should be more accessible 

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1072]
Originally I thought this could be addressed with #1059 but especially since the contents of the slider is so varied this should be addressed on its own.  

Here are some slider specific guidelines: https://www.w3.org/WAI/tutorials/carousels/
